packages/legacy/core/App/contexts/configuration.tsx

```diff
@@ -8,6 +8,7 @@ import { EmptyListProps } from '../components/misc/EmptyList'
 import { RecordProps } from '../components/record/Record'
 import { Locales } from '../localization'
 import OnboardingPages from '../screens/OnboardingPages'
+import { HelpEntry } from '../types/help'
```

Comment:
Can this be more descriptive? It's specifically for how to acquire a credential - no?
- GetCredentialHelp
- CredentialInstructions
- etc

Reply:
Good point, changed